en/developer_guide/external_media_domain.rst

```diff
@@ -0,0 +1,25 @@
+Adding an external media domain
+===============================
+
+Configuration
+~~~~~~~~~~~~~
+
+Sometimes, developers want an external media domain to improve websites performances.
```

Comment:
I would rather say a word about performance instead of putting the developers will in first